calico-cloud/reference/resources/runtimesecurity.mdx

```diff
@@ -0,0 +1,109 @@
+---
+description: API for this Calico Cloud resource 
+---
+
+# RuntimeSecurity
+
+The **RuntimeSecurity** custom resource (CR) is used to enable and configure Container Threat Detection in a Calico Cloud managed cluster.
+
+### Resource Definition
+
+```yaml
+apiVersion: operator.tigera.io/v1
+kind: RuntimeSecurity
+metadata:
+  name: defaultmy-runtimesecurity-config
+spec:
+  detectorConfig:
+    - id: 190
+      disabled: false
+    - id: 330
+      disbaled: True
+  runtimeExceptionList:
+    - matching: regex
+      processInvocation: "/bin/ls*"
+      pod: "not-evil-pod"
+      namespace: "default"
+    - matching: exact
+      pod: "nginx"
+      namespace: default
+    - matching:
+      namespace: "company-operations"
+```
+
+## Runtime Security Definition 
+
+### Metadata
+
+| Field  | Description                                   | Accepted Values                           | Schema |
+| ------ | --------------------------------------------- | ----------------------------------------- | ------ |
+| name   | The name of the runtime security resource.    | Lower-case alphanumeric with optional `-` | string |
+| labels | A set of labels to apply to this resource.    |                                           | map    |
+
+
+### Spec
+
+| Field                    | Description                                                                                  | Accepted Values  | Schema                                                | Default |
+| ------------------------ | -------------------------------------------------------------------------------------------- | ---------------- | ----------------------------------------------------- | ------- |
+| detectorConfig           | Configuration parameters for a detector in a Calico Cloud managed cluster.                   |                  | [DetectorConfig](#detectorconfig)                     |         |
```

Comment:
Can we say "Configuration that allows particular threat detectors to be disabled".

Reply:
(I understand we might add more function here in the future.  When we do that, we can update the doc accordingly.  For now, the only thing this does is disabling, so let's say that.)